src/sentry/sentry_metrics/client/kafka.py

```diff
@@ -50,12 +50,10 @@ def get_retention_from_org_id(org_id: int) -> int:
 
 class KafkaMetricsBackend(GenericMetricsBackend):
     def __init__(self) -> None:
-        kafka_topic_name = settings.KAFKA_INGEST_PERFORMANCE_METRICS
-        self.kafka_topic = Topic(kafka_topic_name)
-
-        kafka_topic_dict = settings.KAFKA_TOPICS[kafka_topic_name]
-        assert kafka_topic_dict is not None
-        cluster_name = kafka_topic_dict["cluster"]
+        logical_topic = Topic.INGEST_PERFORMANCE_METRICS
```

Comment:
can we make this `Topic` enum a basic class with strings? some of the tests fail now because an enum variant is not an actual string

Reply:
Damn, the failures are a real bug. Looks like there are more pre-requisites before I can merge this (specifically, all the consumer definitions need to use the new way)